CMSgov/mint-app
# [MINT 3040]/additional details patient protection

# MINT-3040

## Description

- Add additional fields for Patient Protection option for funding source and funding source R

## How to test this change

1.  Make sure the new fields render as expected in the ticket
2. Make sure read only, csv, change history all render as expected.
3. Make sure all new unit tests pass

If desired, you can use the updated postman query to assert the field functions as expected from a backend perspective.

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
